packaging/nnstreamer.spec

```diff
@@ -564,6 +568,7 @@ cp -r result %{buildroot}%{_datadir}/nnstreamer/unittest/
 %{_prefix}/lib/nnstreamer/decoders/libnnstreamer_decoder_*.so
 %{_prefix}/lib/nnstreamer/converters/libnnstreamer_converter_*.so
 %{_prefix}/lib/nnstreamer/filters/libnnstreamer_filter_cpp.so
+%{_libdir}/libnnstreamer_protobuf.so
```

Comment:
Don't you need ```%if 0%{?protobuf_support}``` for this file?

Reply:
Right. An error occurs if there is no protobuf. (line 569 also) I will add condition.
Would it be better to make protobuf packages or just add condition?

Reply:
Let's make it a subpackage because this incurs additional install-time dependency (protobuf) and there might be a Tizen device that does not have protobuf installed.

Reply:
I added it as the subpackage. I will change the same contents to the flatbuf in another PR.